etest/tests.py

```diff
@@ -104,13 +104,13 @@ def run(self):
         for command in self.commands:
             container_name = str(uuid.uuid4())
 
-            docker.container.create(
-                image=image_name,
-                name=container_name,
-                environment=self.environment,
-                volumes=[
-                    "/overlay",
-                    "/usr/portage",
+            container = docker.container.create(
```

Comment:
This doesn't seem to match black's formatting.  Can you ensure you've got pre-commit installed and running locally?

Reply:
pre-commit doesnt mark it. I'll still change it, but we might want to look at that

Reply:
pre-commit isnt complaining. What exactly is faulty?

Reply:
Nothing anymore.  This has been resolved.  Previously the assignments were spaced and single quotes were in use.